src/locales/de/translation.json

```
@@ -124,6 +131,7 @@
   "Terrain exaggeration": "Gelände-Übertreibung",
```

Verrry literal translations into german are .. akward.

```suggestion
  "Terrain exaggeration": "Geländeüberhöhung",
```